packages/client/src/components/InboxMessageAdmin/Inbox.component.js

```diff
@@ -3,24 +3,34 @@ import React, { useEffect, useState } from 'react';
 import './Inbox.style.css';
 import { Link } from 'react-router-dom';
 import getApiBaseUrl from '../../utils/getApiBaseURL';
+import { UserAuth } from '../../firebase/AuthContext';
 
 export const Inbox = () => {
+  const { user } = UserAuth();
   const [messages, setMessages] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
-  function getMessages() {
-    const promise = fetch(`${getApiBaseUrl()}/api/contactUsForm`).then(
-      (response) => response.json(),
-    );
-    return promise;
-  }
   useEffect(() => {
+    function getMessages() {
+      const promise = fetch(`${getApiBaseUrl()}/admin/inbox-admin`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          authorization: `Bearer ${user.accessToken}`,
+        },
+      }).then((response) => response.json());
+      return promise;
+    }
```

Comment:
```suggestion

    const getMessages = () => {
      const promise = fetch(`${getApiBaseUrl()}/admin/inbox-admin`, {
        method: 'GET',
        headers: {
          'Content-Type': 'application/json',
          authorization: `Bearer ${user.accessToken}`,
        },
      }).then((response) => response.json());
      return promise;
    }
    
  useEffect(() => {
```

If this does not break the functionality, I would suggest moving the function out, so that the useEffect does what it needs to do, without also having a function defined inside of it. Cleaner, more structured, and more readable.